test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -71,10 +71,10 @@ Describe "ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs)"
 		$functionDefinition>$functionDefinitionFile
 
         $PowerShell = [powershell]::Create()
+        $PowerShell.AddScript(". $functionDefinitionFile").Invoke()

```

Comment:
Sorry. Now I measure-command `ExecuteCommand` before and after patch - difference is ~8 ms on my notebook. So 8 ms * 84 = 676 ms. I don't measure memory consumption. 

Now I am surprised why two parser tests failed 😕